key.core/src/main/java/de/uka/ilkd/key/speclang/jml/translation/JMLSpecFactory.java

```diff
@@ -926,10 +926,13 @@ public ImmutableSet<Contract> createFunctionalOperationContracts(String name, IP
             result = result.add(contract);
         } else {
             // create two contracts for each diamond and box modality
+            Map<LocationVariable, Term> boxPres = new LinkedHashMap<>(pres);
```

Comment:
Shouldn't this be a copy of `clauses.requires`? Previously that map was used when constructing the FunctionalOperationContract.